mbed/mbed.py

```diff
@@ -37,7 +37,7 @@
 
 
 # Application version
-ver = '1.4.0'
+ver = '1.5.0'
```

Comment:
Seems that the rebase picked other changes as well. Please rebase correctly so this picks only the changes against the current mbed CLI master.